include/ntop_defines.h

```diff
@@ -756,6 +756,8 @@
 #define HOST_POOL_MEMBERS_KEY NTOPNG_PREFS_PREFIX ".host_pools.members.%s"
 #define HOST_POOL_SHAPERS_KEY NTOPNG_PREFS_PREFIX ".%u.l7_policies.%s"
 #define HOST_POOL_DETAILS_KEY NTOPNG_PREFS_PREFIX ".host_pools.details.%u"
+#define HOST_POOL_STATS_KEY NTOPNG_PREFS_PREFIX ".host_pools.stats.ifid_%u.pool_%u"
+#define HOST_POOL_LAST_CHECK_DATE_KEY NTOPNG_PREFS_PREFIX ".host_pools.last_check_day"
```

Comment:
The macro name contains "DATE", the actual key name contains "day", then in the code it is sometime compared with a month. Please make this more clear (I suggest to use "epoch" everywhere).